### ISSUE TITLE
False positive for private constructor in `unicorn/consistent-class-member-order`

### ISSUE THREAD

**Issue by crystalfp** [sindresorhus/eslint-plugin-unicorn#3260]
The rule complains that "Expected private method to come before public method" when a private constructor is the first method in the class. If the constructor is not private everything is OK. A private constructor is normally found in a singleton class.

unicorn/consistent-class-member-order from     "eslint-plugin-unicorn": "^66.0.0",

```js
class Singleton {
    private static instance: Singleton;
    private constructor() {}
    onePublicMethod() {}
    static getInstance(): Singleton {

        if(!Singleton.instance) {
            Singleton.instance = new Singleton();
       	}

        return Singleton.instance;
   }
}
```

**Comment by sindresorhus** [comment]
The private constructor isn't actually the cause here: a constructor is always classified as `constructor` regardless of its accessibility. What's being flagged is `getInstance` being a `static` method placed last, since static methods are ordered near the top.

The real problem was that the message only named the *groups*, which made it easy to blame the wrong member. I will get the message fixed.
